src/bot/api.rs

```diff
@@ -515,6 +600,7 @@ impl Bot {
         Q: Into<String>,
         O: Into<Vec<String>>,
     {
+        // FIXME: parse_mode
```

Comment:
https://core.telegram.org/bots/api#sendpoll there are not parse_mode

Reply:
`explanation_parse_mode`